static/css/styles.css

```diff
@@ -221,6 +221,59 @@ footer {
     cursor: pointer;
 }
 
+/* Tooltip Styling */
```

Comment:
I removed this because we got rid of the tooltips, is there a reason we want to add the styling back in? Just want to minimize any unnecessary css code :)

Reply:
deleted! must have added it accidentally back through merging.